File: Seção 2/Aula 14/JavaScript/app2.js
function isAdulthood(yearOfBirth) {
  return new Date().getFullYear() - yearOfBirth >= 18;
}

function main() {
  let yearOfBirth = 2000;

  console.log(`A pessoa é adulta: ${isAdulthood(yearOfBirth)}`);

  yearOfBirth = 2015;

  console.log(`A pessoa é adulta: ${isAdulthood(yearOfBirth)}`);
}

main();

File: Seção 2/Aula 14/Python/app2.py
from datetime import datetime


def is_adulthood(year_of_birth):
    return datetime.now().year - year_of_birth >= 18


def main():
    year_of_birth = 2000

    print(f"A pessoa é adulta: {is_adulthood(year_of_birth)}")

    year_of_birth = 2015

    print(f"A pessoa é adulta: {is_adulthood(year_of_birth)}")


if __name__ == "__main__":
    main()
